File: adhoc/carolinacigerza/1962.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
         Scanner teclado = new Scanner(System.in);
         int n, t, i, ano;    
         n = teclado.nextInt();
         String anos[] = new String[n];    
         for (i = 0; i < n; i++) {
              t = teclado.nextInt();
              if (t > 2015) {
                ano = t - 2014;
                anos[i] = ano + " A.C.";
              } 
              else if (t < 2015) {
                ano = 2015 - t;
                anos[i] = ano + " D.C.";
              }
              else {
                anos[i] = "1 A.C.";
              }      
         }
         for (i = 0; i < n; i++) {
             System.out.println(anos[i]);
         }
    }
}

File: adhoc/jeoftoncosta/1962.py
N = int(input())
anos = []
for i in range(N):
    i = int(input())
    anos.append(i)

for ano in anos:
    if ano>2015:
        T = (ano - 2015)+1
        print(T,"A.C.")
    elif ano<2015:
        T = (ano - 2015)*(-1)
        print(T,"D.C.")
    else:
        T= (ano-2015)+1
        print(T,"A.C.")